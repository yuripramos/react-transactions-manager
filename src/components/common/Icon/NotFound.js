import React from "react";
/*eslint-disable*/
function NotFound() {
  return (
    <svg
      xmlns="http://www.w3.org/2000/svg"
      width="150"
      height="150"
      viewBox="0 0 150 150"
    >
      <g fill="#374654" fillRule="nonzero" stroke="#F6F6F6">
        <path
          strokeLinecap="round"
          strokeWidth="6"
          d="M134.734 3.41H16.118C9.099 3.41 3.409 9.1 3.409 16.117v118.616c0 7.019
          5.69 12.71 12.709 12.71h118.616c7.019 0 12.71-5.691
          12.71-12.71V16.118c0-7.019-5.691-12.709-12.71-12.709zm0
          131.324H16.118V50.008h118.616v84.726zm0-93.198H16.118V16.118h118.616v25.418z"
        />
        <path d="M83.523 24.716h8.523v8.523h-8.523zM100.568
        24.716h8.523v8.523h-8.523zM117.614 24.716h8.523v8.523h-8.523zM41.761 101.42h8.523v8.523h8.523v-34.09h-8.523v17.045h-8.523V75.852H33.24v17.046c0 4.7 3.823 8.522 8.522 8.522zM100.568 101.42h8.523v8.523h8.523v-34.09h-8.523v17.045h-8.523V75.852h-8.523v17.046c0 4.7
        3.824 8.522 8.523 8.522zM75.852 109.943c7.05 0 12.784-5.734 12.784-12.784v-8.523c0-7.05-5.734-12.784-12.784-12.784-7.05 0-12.784 5.734-12.784 12.784v8.523c0 7.05 5.734 12.784 12.784 12.784zm-4.261-21.307a4.267 4.267 0 0 1 4.261-4.261 4.267 4.267 0 0 1 4.262 4.261v8.523a4.267 4.267
        0 0 1-4.262 4.261 4.267 4.267 0 0 1-4.261-4.26v-8.524z" />
      </g>
    </svg>
  );
}

export default NotFound;
