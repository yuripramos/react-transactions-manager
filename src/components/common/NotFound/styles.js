import styled, { css } from "styled-components";
import { rem, media } from "../../../styles/tools";

export const Content = styled.div`
  margin-top: ${rem(33)};

  a {
    width: auto;
  }

  ${media.md(css`
    margin-top: ${rem(50)};
  `)};
`;

export const Wrapper = styled.div`
  svg {
    height: ${rem(75)};
    width: auto;

    ${media.md(css`
      height: auto;
    `)};
  }
`;
