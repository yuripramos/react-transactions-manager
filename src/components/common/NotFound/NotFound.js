import React from "react";
import { black30 } from "../../../styles/settings";
import Icon from "../Icon";
import DefaultContent from "../DefaultContent";

import { Content, Wrapper } from "./styles";

function NotFound() {
  return (
    <Wrapper>
      <DefaultContent
        data-test="NotFound"
        Icon={() => <Icon name="NotFound" color={black30} />}
        primaryText={`Page Not Found`}
        secondaryTexts={["Sorry Page Not Found"]}
      >
        <Content />
      </DefaultContent>
    </Wrapper>
  );
}

export default NotFound;
