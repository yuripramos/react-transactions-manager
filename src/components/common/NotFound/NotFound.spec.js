import React from "react";
import { shallow } from "enzyme";

import NotFound from "./NotFound";

describe("NotFound component", () => {
  it("should match snapshot", () => {
    expect(shallow(<NotFound />)).toMatchSnapshot();
  });
});
