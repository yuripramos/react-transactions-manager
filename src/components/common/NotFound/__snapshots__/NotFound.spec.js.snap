// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component should match snapshot 1`] = `
<styled.div>
  <DefaultContent
    Icon={[Function]}
    data-test="NotFound"
    primaryText="Page Not Found"
    secondaryTexts={
      Array [
        "Sorry Page Not Found",
      ]
    }
  >
    <styled.div />
  </DefaultContent>
</styled.div>
`;
